### DISCUSSION TITLE
Unable to update data using supabase-py

### DISCUSSION THREAD

**Discussion by musaibahmedrazzaqui** [supabase/supabase#22008] [Questions]
Hi everyone, im able to successfully retrieve supabase data in python using the select command
`supabase.table('verses').select('content').eq('audio_file', '1 John_Chapter 1_Verse 3').execute()`
But while updating the same i get the error. 
Here's my line of code that executes the update query:

`supabase.table('verses').update({'content':'This is a test content'}).eq('audio_file', '1 John_Chapter 1_Verse 3').execute()`

The error im getting is 
{'data': {'code': 'PGRST102', 'details': None, 'hint': None, 'message': 'Empty or invalid json'}, 'status_code': 400}

The data type for the content field is text

**Comment by silentworks** [comment]
Can you confirm you are using the `supabase` library and not the `supabase=py` library? we don't maintain or have access to the `supabase-py` library and only push new updates to the `supabase` library on [PyPi](https://pypi.org/project/supabase/).
